packages/expo-secure-store/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### 🎉 New features
 
 ### 🐛 Bug fixes
+- Changed `requireAuthentication` option to also require biometrics on iOS (matches Android behavior) ([#18591](https://github.com/expo/expo/pull/18591) by [@stefan-schweiger](https://github.com/stefan-schweiger))
```

Comment:
```suggestion

### ⚠️ Notices

- Changed `requireAuthentication` option to also require biometrics on iOS (matches Android behavior) ([#18591](https://github.com/expo/expo/pull/18591) by [@stefan-schweiger](https://github.com/stefan-schweiger))
```

since it's a behavior change, i would like to put this changelog into notice section.